jane/doc/extensions/_04-modes/intro.md

```diff
@@ -183,5 +183,64 @@ they are once.
 More documentation on uniqueness and linearity is available
 [here](../../uniqueness/intro/).
 
-<!-- CR ccasinghino: Sections needed for statefulness, visibility, and yielding -->
+## Future modes: Yielding
 
+|----------------|
+| yielding       |
+| `|`            |
+| **unyielding** |
+{: .table }
+
+Yielding is a future axis that tracks whether a function is permitted to perform
+effects that will be handled in its parent stack. See [the OCaml Manual entry
+for effect handlers](https://ocaml.org/manual/5.3/effects.html).
+
+Yielding has different defaults depending on the locality axis: *global* values are
+defaulted to *unyielding*, while *local* values are defaulted to *yielding*.
+More documentation on mode implications is available [here](../_05-kinds/syntax.md).
+
+Yielding is irrelevant for types that do not contain functions, and values of such types
+*mode cross* on the yielding axis; they may be used as unyielding even
+when they are yielding.
+
+# Past modes: Visibility
```

Comment:
I think maybe there should be a new heading about purity? And then make these subheadings, echoing the structure above.

Reply:
Fixed.

Reply:
{{resolve}}